FCL edits / review (#1)
10thfloor/fcl-docs: README.md

## Address replacement in scripts and transactions

> Configuration keys that start with `0x` will be replaced in Cadence scripts and transactions input to FCL. Typically this is used to represent account addresses. Account addresses for the same contract will be different depending on the Flow network you're interacting with (eg. Testnet, Mainnet). 

Author: ```suggestion
Configuration keys that start with `0x` will be used to find-and-replace their values in Cadence scripts and transactions input to FCL. Typically this is used to represent account addresses. Account addresses for the same contract will be different depending on the Flow network you're interacting with (eg. Testnet, Mainnet). 
```

> :warning: **This method can only be used in web browsers.**

> Authenticate the current user via any wallet that supports FCL. Once called, FCL will initiate communication with the configured `discovery.wallet` endpoint which lets the user select a wallet. Once the wallet provider has authenticated the user, FCL will set the values on the [current user](##`CurrentUserObject`) object.

Author: ```suggestion
Calling this method will authenticate the current user via any wallet that supports FCL. Once called, FCL will initiate communication with the configured `discovery.wallet` endpoint which lets the user select a wallet. Once the wallet provider has authenticated the user, FCL will set the values on the [current user](##`CurrentUserObject`) object.
```

## `fcl.currentUser().subscribe(callback)`

> Subscribe to changes in the currently authorized user. The callback will be called when the user authenticates and un-authenticates, making it easy to update the UI accordingly.

Author: ```suggestion
The callback passed to subscribe will be called when the user authenticates and un-authenticates, making it easy to update the UI accordingly.
```

## `fcl.send([...builders])`

Sends arbitrary scripts, transactions, and requests to Flow.

> This method onsumes an array of [builders](https://google.ca) that are to be resolved and sent. The builders required to be included in the array depend on the [interaction](##`Interactions`) that is being built.

Author: ```suggestion
This method consumes an array of [builders](https://google.ca) that are to be resolved and sent. The builders required to be included in the array depend on the [interaction](##`Interactions`) that is being built.
```

Author: typo.

## `fcl.decode(response)`

> Decodes the response from `fcl.send()` into the appropriate JSON representation of all relevant keys and values returned from Cadence code.

Author: ```suggestion
Decodes the response from `fcl.send()` into the appropriate JSON representation of any values returned from Cadence code.
```
